configs.js

```diff
@@ -0,0 +1,42 @@
+let alert = document.getElementById('alert');
```

Comment:
Alert is a very common name so use something else or indicate clearly on why we need this alert ?  

OR 

```(function() {})()``` maybe scope the whole code in an IIFE ? 

what do you think ?